### PR TITLE
Added fingerprint prefix in order to match the one on strapi website

### PR DESCRIPTION
**Pull request by ivanThePleasant**
### What does it do?

This adds a prefix to the `deviceId` property on the telemetry call so that it can be matched later on amplitude. Same prefix has been added to the strapi website analytics.

### Why is it needed?

Prefix is needed so that it would be possible to exclude certain data from amplitude analytics.

### How to test it?

N/A

### Related issue(s)/PR(s)

[Let us know if this is related to any issue/pull request](https://github.com/strapi/website-2020/pull/853)

### CODE REVIEW

**Review by Mcastres — DISMISSED** [review]
LGTM

**Comment by alexandrebodin** [comment]
Commented in the website issue before we can move forward. 

I only see that changing the deviceId only for this event would prevent linking it with the rest of the events sent, which doesn't seem to be the goal.

**Comment by ivanThePleasant** [comment]
@alexandrebodin noted, also replied on the website pr. Please let me know if that reply sufficiently explains the matter. We only need to link these two events because both are generated on the front-ends by fingerprint.js and are used to track the conversion rate between website visits and admin panel initialisations.

File: packages/core/admin/admin/src/pages/App/index.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function App() {
               body: JSON.stringify({
                 event: 'didInitializeAdministration',
                 uuid,
-                deviceId,
+                deviceId: `web-fingerprint-${deviceId}`,
```
Comment (alexandrebodin) [inline packages/core/admin/admin/src/pages/App/index.js:80]: Can we just change the getUID function to add the prefix instead. No need to leak this abstraction in the code here

**Review by alexandrebodin — CHANGES_REQUESTED** [review]

File: packages/core/admin/admin/src/pages/App/index.js (hunk 1 above)
Comment (ivanThePleasant) [reply packages/core/admin/admin/src/pages/App/index.js:80]: No worries, I'll rework this 👍

**Review by alexandrebodin — APPROVED** [review]
LGTM

**Pull request merged**